### PR TITLE
add step to remove old version packages 

### PR DESCRIPTION
**Pull request by dmsolr**

### CODE REVIEW

File: release.md [hunk 1]
```diff
@@ -100,6 +100,7 @@ Move the release from RC folder to the dist folder. This will begin the file syn
 ....
 enter your apache password
 ....
+> svn delete -m "remove old version" https://dist.apache.org/repos/dist/dev/skywalking/kong/x.y.z
```
Comment (wu-sheng) [inline release.md]: You should not do this here. Because your website is targeting the mirror and download links.

Comment (wu-sheng) [reply release.md]: You have to make sure all things ready and remove the old release, and it is not x.y.z, it is the previous release version.

Comment (dmsolr) [reply release.md]: I am sorry for my incorrect title and comment. I am going to delete dev directory once the packages had moved to release directory.

Comment (wu-sheng) [reply release.md]: Zhenxu's point is, removing the previous release. Such as removing 0.1.0 once the 0.2.0 is released. Not the thing in dev.

Comment (wu-sheng) [reply release.md]: ` svn mv ` already removed the package in the dev folder.

Comment (dmsolr) [reply release.md]: Got it. Fixed

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**